File: javascript/leap/leap.js
var Year = function(inputYear) {
    this.year = inputYear;
};

Year.prototype.isLeap = function() {
    if (this.year % 100 === 0 && this.year % 400 !== 0) {
        return false;
    } else if (this.year % 4 === 0) {
        return true;
    }else {
        return false;
    }
};

module.exports = Year;

File: csharp/Leap/Leap.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Application
{
    public class Year
    {
        public bool IsLeap(int year)
        {
            if (year % 4 == 0)
            {
                return true;
            } else 
            {
                return false;
            }
        }
    }
}